docs/src/consts.ts

```diff
@@ -1,2 +1,2 @@
-export const MAX_CONTENT_WIDTH = "80rem";
+export const MAX_CONTENT_WIDTH = "90rem";
```

Comment:
Keep it as it was before. You will not need to do calculations then ;)

Reply:
Justified above ☝️